### PR TITLE
couple small fixes

### PR DESCRIPTION
**Pull request by tzemanovic**
Hi, this is a fix for conditional compilation in the transport mod with couple small fixes.

### CODE REVIEW

File: lib/Cargo.toml [hunk 1]
```diff
@@ -29,7 +29,7 @@ default = [ "transport_usb", "transport_tcp", "transport_ble", "transport_usb_li
 
 thiserror = "1.0.40"
 encdec = "0.9.0"
-ledger-proto = { version = "0.1.0", default_features = false, features = [ "std" ] }
+ledger-proto = { version = "0.1.0", path = "../proto", default-features = false, features = [ "std" ] }
```
Comment (tzemanovic) [inline lib/Cargo.toml]: This no longer compiles with 0.1.0, will need a new version. I added `path` that's used locally

Comment (ryankurte) [reply lib/Cargo.toml]: hmm what's not working here? it shouldn't be upset about the cargo tag changes without `edition = "2024"`?

Comment (tzemanovic) [reply lib/Cargo.toml]: sry for the vague note - I don't recall and it seems to compile just fine

**Comment by ryankurte** [comment]
thanks for the contribution! i've landed #8 (because it was open earlier), happy to take the rest without the `path = "../proto"` change if you're up for a rebase?

**Comment by tzemanovic** [comment]
rebased, thank you too!

**Review by ryankurte — APPROVED** [review]

**Comment by tzemanovic** [comment]
hi, could you pls publish a new release with the fixes on crates.io?

**Pull request merged**